receiver/awsecscontainermetricsreceiver/internal/awsecscontainermetrics/resource.go

```diff
@@ -21,16 +21,20 @@ import (
 	conventions "go.opentelemetry.io/collector/model/semconv/v1.5.0"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/ecsutil"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/docker"
 )
 
 func containerResource(cm ecsutil.ContainerMetadata) pdata.Resource {
 	resource := pdata.NewResource()
+
+	imageName, imageTag, _ := docker.ImageToElements(cm.Image)
```

Comment:
I'm thinking that silently dropping both fields in case of error is not ideal (even if the chance of error is pretty low). We probably can log a debug line here. And I think we should use `cm.Image` as `imageName` in case of error. It applies to both k8scluster and this receivers.

Reply:
Adding adding a additional log here make sense. 

Regarding the fall back to `cm.Image`, I think it is better to set it on empty string than to add the raw image in there.     
My reasoning is that this can cause errors in downstream systems which do not expect this as it is not defined in the specification.

Reply:
Ok. It sounds reasonable to me. Let's see it other folks think about it